apps/system/js/nfc_handover_manager.js

```diff
@@ -238,7 +238,12 @@
           this.settingsNotified = true;
         }
       } else {
-        action.callback.apply(this, action.args);
+        Service.request('Bluetooth:adapter').then((adapter) => {
+          this._adapter = adapter;
+          action.callback.apply(this, action.args);
```

Comment:
all `_doXxxx` methods are triggered after `_doAction`, so we can safely remove all adapter checkers in those methods.